app/pilot-program/page.tsx

```diff
@@ -34,7 +34,7 @@ export default function PilotProgramPage() {
     <main>
       {/* <!-- hero --> */}
       <div className="bg-white">
-        <section className="w-full bg-top bg-cover bg-no-repeat bg-[url('/img/pilot/hero.jpg')]">
+        <section className="w-full bg-top bg-cover bg-no-repeat bg-white md:bg-[url('/img/pilot/hero.jpg')]">
```

Comment:
On all pages, don't use an image background for small sizes. Makes text more legible.